setuptools/_distutils/compilers/C/msvc.py

```diff
@@ -255,10 +257,10 @@ class Compiler(base.Compiler):
     obj_extension = '.obj'
     static_lib_extension = '.lib'
     shared_lib_extension = '.dll'
-    static_lib_format = shared_lib_format = '%s%s'
+    static_lib_format = static_lib_format = '%s%s'
```

Comment:
🤔

Reply:
NVM, looks like you are already aware of this https://github.com/python/typeshed/blob/f6e99d20b503225404c638a536c39759b252dbc2/stubs/setuptools/setuptools/_distutils/compilers/C/msvc.pyi#L16